### PR TITLE
[pumpio] Fixes/imports:

### PR DESCRIPTION
**Pull request by Quix0r**
Fixes/imports:
- added curly braces and spaces for better readability
- merged 2 `if()` blocks into one for lesser nested code blocks
- added type-hint `App` and `array` where applyable
- better "import" (`use` under PHP) classes and not reference them directly,
  reducing and improving readability

### CODE REVIEW

File: pumpio/pumpio.php [hunk 1]
```diff
@@ -326,10 +328,10 @@ function pumpio_settings(&$a, &$s)
 	$s .= '<div class="settings-submit-wrapper" ><input type="submit" id="pumpio-submit" name="pumpio-submit" class="settings-submit" value="' . L10n::t('Save Settings') . '" /></div></div>';
 }
 
-function pumpio_settings_post(&$a, &$b)
+function pumpio_settings_post(App $a, array &$b)
 {
-	if (x($_POST, 'pumpio-submit')) {
-		if (x($_POST, 'pumpio_delete')) {
+	if (x($_POST,'pumpio-submit')) {
+		if (x($_POST,'pumpio_delete')) {
```
Comment (MrPetovan) [inline pumpio/pumpio.php]: Why did you remove the spaces in the two previous lines? Please keep them as is.

Comment (Quix0r) [reply pumpio/pumpio.php]: Ops? My fault. Okay, I revert them. And I can switch here away from `x()` (deprecated) to `!empty()` okay?

Comment (MrPetovan) [reply pumpio/pumpio.php]: By all means!

**Comment by Quix0r** [comment]
Done.

**Comment by MrPetovan** [comment]
I appreciate the touch, but please don't mention my GitHub handle in your commit messages. It creates additional unactionable notifications when you commit and when we merge.
